File: Python/PRAK605-2210817310017-Rohid Akmal.py
ordo = int(input())
matrik1 = []
print("Matriks A")
for i in range(0, ordo):
    isi1 = list(map(int, input().split()))
    matrik1.append(isi1[0: ordo])
matrik2 = []
print("Matriks B")
for i in range(0, ordo):
    isi2 = list(map(int, input().split()))
    matrik2.append(isi2[0: ordo])
print("\nMatriks A x B")
for i in range(0, ordo):
    for j in range(0, ordo):
        hsl = 0
        for k in range(0, ordo):
            hsl += matrik1[i][k] * matrik2[k][j]
        print(hsl, end=' ')
    print()

File: C/PRAK605-2210817310017-Rohid_Akmal.c
#include <stdio.h>

void main(){
    int ordo, x, y, z, isi, hsl;
    scanf("%d", &ordo);
    int matrik1[ordo][ordo], matrik2[ordo][ordo];
    printf("\nMatriks A\n");
    for(x = 0; x < ordo; x++){
        for(y = 0; y < ordo; y++){
            scanf("%d", &isi);
            matrik1[x][y] = isi;}}
    printf("\nMatriks B\n");
    for(x = 0; x < ordo; x++){
        for(y = 0; y < ordo; y++){
            scanf("%d", &isi);
            matrik2[x][y] = isi;}}
    printf("\nMatriks A x B\n");
    for(x = 0; x < ordo; x++){
        for(y = 0; y < ordo; y++){
            for(z = 0, hsl = 0; z < ordo; z++){
                hsl += matrik1[x][z] * matrik2[z][y];}
            printf("%d ", hsl);}
    printf("\n");}
}